cli/kaos_cli/services/tests/test_state_service.py

```diff
@@ -0,0 +1,228 @@
+import os
+import time
+from parameterized import parameterized
+from configobj import ConfigObj
+
+from unittest import TestCase
+
+from kaos_cli.services.state_service import StateService
+from kaos_cli.constants import CONFIG_PATH, KAOS_STATE_DIR
+
+
+def state_service_test_inputs():
+    return [
+        (
+            'context1', 'param1'
+        ),
+        (
+            'context2', 'param2'
+        ),
+        (
+            '', 'param3'
+        ),
+        (
+            'context4', ''
+        ),
+        (
+            '', ''
+        ),
+        (
+            {}, 'param6'
+        ),
+        (
+            'context7', {}
+        ),
+        (
+            {}, {}
+        ),
+        (
+            '', {}
+        ),
+        (
+            {}, ''
+        ),
+        (
+            None, 'param11'
+        ),
+        (
+            'context11', None
+        ),
+        (
+            None, None
+        ),
+        (
+            {}, None
+        ),
+        (
+            None, {}
+        ),
+        (
+            '', None
+        ),
+        (
+            None, ''
+        ),
+        (
+            1, 'param18'
+        ),
+        (
+            'context', 2
+        ),
+        (
+            3, {}
+        ),
+        (
+            {}, 4
+        ),
+        (
+            5, ''
+        ),
+        (
+            '', 6
+        ),
+        (
+            7, None
+        ),
+        (
+            None, 8
+        ),
+
+    ]
+
+
+class TestStateService(TestCase):
+
+    # Arrange
+    TestCase.state_service = StateService()
+    TestCase.switch = False
+
+    def test_state_service_create(self):
+
+        # Act
+        TestCase.state_service.create()
+
+        # Assert
+        self.assertTrue(os.path.exists(KAOS_STATE_DIR))
+
+    def test_state_service_is_created(self):
+
+        # Act
+        TestCase.state_service.is_created(KAOS_STATE_DIR)
+
+        # Assert
+        self.assertEqual(True, TestCase.state_service.is_created(KAOS_STATE_DIR))
```

Comment:
You're acting out the test twice, maybe name the returned value from the `is_created` method and then assert it's equal to `True`? For example:

```python
is_created = TestCase.state_service.is_created(KAOS_STATE_DIR)
self.assertEqual(True, is_created)
```

Reply:
True! As suggested, this value is now stored as a variable which later gets asserted.